app/concerns/persistable.rb

```diff
@@ -1,6 +1,15 @@
 module Persistable
 
   module ClassMethods
+    def attributes
+      self::ATTRIBUTES
```

Comment:
by using `self::ATTRIBUTES` I'm scoping the access of the `ATTRIBUTES` constant to be scoped to whatever `self` in the context of this method and not module is.